src/HDF5.jl

```diff
@@ -1790,9 +1789,9 @@ function h5a_write(attr_id::hid_t, memtype_id::hid_t, v::VLen{T}) where {T<:Unio
     vp = vlenpack(v)
     h5a_write(attr_id, memtype_id, vp)
 end
-h5a_create(loc_id::hid_t, name::String, type_id::hid_t, space_id::hid_t) = h5a_create(loc_id, name, type_id, space_id, _attr_properties(name), H5P_DEFAULT)
+h5a_create(loc_id, name, type_id, space_id) = h5a_create(loc_id, name, type_id, space_id, H5P_DEFAULT, H5P_DEFAULT) # last two arguments not used in HDF5 lib
```

Comment:
https://confluence.hdfgroup.org/display/HDF5/H5A_CREATE2 says that the argument should be `H5P_DEFAULT`, not sure why `_attr_properties(name)` was set here.